Userland/Utilities/ls.cpp

```diff
@@ -32,10 +32,18 @@
 #include <time.h>
 #include <unistd.h>
 
+struct FileMetadata {
+    String name;
+    String path;
+    struct stat stat;
```

Comment:
Instead of the `memset` below, I suggest simply initializing this explicitly:
```cpp
struct stat stat {};
```

Reply:
I've removed the calls to `memset()`. Maybe I went overboard with explicit constructors, I'm not sure what are the precise C++ rules governing that particular case...